### ISSUE TITLE
Block commands repeating 

### ISSUE THREAD

**Issue by savnith** [TotalFreedom/TotalFreedomMod#2074]
PR Suggestion: Block players from repeating there last sent command.

**Comment by lemonsked** [comment]
I don't think this is needed. Sometimes repeating commands is useful, and when/if they spam they can just be smitten by an admin.
